.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 coverage
+package-lock.json
```

Comment:
Any specific reason we're ignoring this file now?

Reply:
This is a module, if we have that file we need to update this module whenever a dependency has a patch release.